roadmap/implementers-guide/src/messaging.md

```diff
@@ -36,7 +36,9 @@ The weight that processing of the dispatchables can consume is limited by a prec
 that some dispatchables will be left for later blocks. To make the dispatching more fair, the queues are processed turn-by-turn
 in a round robin fashion.
 
-Other kinds of upward messages can be introduced in the future as well. Most likely the HRMP channel management.
+Other kinds of upward messages can be introduced in the future as well. Potential candidates are channel management for
+horizontal message passing (XCMP and HRMP, both are to be described later), new validation code signalling, or other
```

Comment:
"Later" as in at a later point of time in this document, or as in further down in this document, if so, a link might be helpful

Reply:
Later, is actually as in later PRs : ) I am preparing one as we speak

Reply:
Ah sorry I misinterpreted. Further down the document

Reply:
Addressed